core/modules/filters/getIndex.js

```diff
@@ -0,0 +1,32 @@
+/*\
+title: $:/core/modules/filters/getIndex.js
+type: application/javascript
+module-type: filteroperator
+
+returns the value at a given index of datatiddlers
+
+\*/
+(function(){
+
+/*jslint node: true, browser: true */
+/*global $tw: false */
+"use strict";
+
+/*
+Export our filter function
+*/
+exports.getIndex = function(source,operator,options) {
+	var data,title,results = [];
+	if(operator.operand){
+		source(function(tiddler,title) {
+			title = tiddler ? tiddler.fields.title : title;
+			data = options.wiki.parseTextReference(title,"",operator.operand,{});
```

Comment:
this doesn't look quite right. `parseTextReference` parses the content of a field or index of a tiddler and returns a parse tree. It does return the source text in the `source` property, but that's really a side effect of the parsing process. It's not an efficient way to access a property index. Better to use `wiki.extractTiddlerDataItem()`.